pykechain/models/activity.py

```diff
@@ -259,8 +259,8 @@ def edit(self, name=None, description=None, start_date=None, due_date=None, assi
 
         if status:
             self._json_data['status'] = str(status)
-        if assignee:
-            self._json_data['assignee'] = assignee
+        if assignees:
+            self._json_data['assignees'] = assignees
```

Comment:
we might consider the add_assignee or remove_assignee methods, but that only when it will come up in the future